File: src/main/java/net/slipp/domain/user/ExistedUserException.java
package net.slipp.domain.user;

public class ExistedUserException extends Exception {
	private static final long serialVersionUID = -989452880795135125L;

	public ExistedUserException() {
		super();
	}

	public ExistedUserException(String arg0, Throwable arg1) {
		super(arg0, arg1);
	}

	public ExistedUserException(String arg0) {
		super(arg0);
	}

	public ExistedUserException(Throwable arg0) {
		super(arg0);
	}
}

File: src/main/scala/net/slipp/domain/user/ExistedUserException.scala
package net.slipp.domain.user

class ExistedUserException(arg0: String, arg1: Throwable) extends Exception(arg0, arg1) {
  def this() = this(null, null)

  def this(arg0: String) = this(arg0, null)

  def this(arg0: Throwable) = this(null, arg0)
}
